iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/ErrorInjectionHelper.java

```diff
@@ -31,7 +31,7 @@
     public static final String FaultCloseReason_Boom = "Boom";
     public static final String FaultCloseReason_Bye = "byebye";
 
-    public static final int DefaultDelayInSec = 1;
```

Comment:
C# SDK uses 5 here, but the service doesn't appear to respect this value. Just in case though, I want to unify with the C# SDK